platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -211,6 +217,17 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 class NvidiaSFPCommon(SfpOptoeBase):
+    platform_sfputil = sonic_platform_base.sonic_sfp.sfputilhelper.SfpUtilHelper()
```

Comment:
Hi Vadym, we requested sdk to provide module based sysfs, we don't use logical port based sysfs here. If SDK did not provide it, please open a feature request for them.